zokrates_core/src/types/conversions.rs

```diff
@@ -268,22 +224,12 @@ mod tests {
             assert_eq!(
                 *unpack.statements.last().unwrap(),
                 FlatStatement::Return(FlatExpressionList {
-                    expressions: (FieldPrime::get_required_bits() - nbits
+                    expressions: (FieldPrime::get_required_bits() - 254
```

Comment:
this is zero, change it to that (legacy from generic bit width we don't need anymore)